### PR TITLE
Swap `belowSteerSpeed` and `resumeRequired` alert priorities

### LINKED ISSUE #28361: `resumeRequired` does not display in UI if `belowSteerSpeed`
On cars without steer to zero, the `belowSteerSpeed` message takes priority over `resumeRequired` when at a standstill. Displaying the latter message at a standstill makes it clear to the driver that their input is required, and its absence may lead to confusion about why the vehicle is not moving.

I have confirmed with several people that this is the case for GM, and am checking on Toyota (those are the only two makes that appear to set the `resumeRequired` message).

### PR DESCRIPTION
**Pull request by nworb-cire**
Should fix https://github.com/commaai/openpilot/issues/28361.

I also added the SnG criterion for sending the resume message so that we don't have to remember to add that if/when we get SnG implemented on GM.

### CODE REVIEW

**Review by adeebshihadeh — CHANGES_REQUESTED** [review]
This is a good fix, but it should use the alert priorities instead.

**Comment by nworb-cire** [comment]
Currently `resumeRequired` is `LOW` and `belowSteerSpeed` is `MID`. Should we set `resumeRequired` to `HIGH` priority, or is it better to swap them? I don't want to clobber distracted driver alerts, although that's perhaps a non-issue while at standstill.

**Comment by adeebshihadeh** [comment]
Unfortunately, the alert priorities are a bit of a mess. They should really be refactored to reason about them more easily, but in the meantime, want to make a simple debug script that prints out the alerts in order of their precedence? 

I'm thinking it would look something like this:
```
State.preEnabled
* HIGHEST: stockFcw, controlsInitializing, startup
* HIGH: stockAeb, steerTempUnavailableSilent
* MID:
* LOW: 
* LOWER: 
* LOWEST: 

State.softDisabling
...
```

Needs to be per-state, since we only [show a subset of alert types](https://github.com/commaai/openpilot/blob/e63e2dde18b9ae2925d1a38c527dccfd0c99d4c5/selfdrive/controls/controlsd.py#L485) in each state.

**Comment by nworb-cire** [comment]
Done. Was a bit unsure of the best way of handling the dynamic alerts, so I just instantiated empty carParams/carState for the time being.

I can clean it up if the output is what you're looking for. Can also split into a separate PR if you'd like.

**Comment by sshane** [comment]
@nworb-cire can you print the events that will now have a higher priority over the minSteerSpeed alert?

**Comment by nworb-cire** [comment]
See #28898

Setting `belowSteerSpeed` to `Priority.LOW` would put it lower in priority than these other `WARNING`s:
```
Priority.HIGH:
       driverDistracted driverUnresponsive speedTooHigh

Priority.MID:
       promptDriverDistracted promptDriverUnresponsive resumeRequired
```

File: selfdrive/car/gm/interface.py
Comment (sshane) [inline selfdrive/car/gm/interface.py:283]: We only need one change, right? Either this or changing the priorities.

Comment (sshane) [reply selfdrive/car/gm/interface.py:283]: I worry that cruiseControl.resume may be noisy, so we will spam the audible min steering speed alert in stop and go.

I haven't personally tried this, so maybe it's a non-issue. Have you been driving on this branch, what is your experience? Is it annoying to hear the alert when you press resume?

**Comment by sshane** [comment]
> Setting belowSteerSpeed to Priority.LOW would put it lower in priority than these other WARNINGs:

> Priority.HIGH:
 >       driverDistracted driverUnresponsive speedTooHigh

> Priority.MID:
 >       promptDriverDistracted promptDriverUnresponsive resumeRequired

Oh wow, we don't alert at all under 7 mph for DM?!

File: selfdrive/car/gm/interface.py
Comment (nworb-cire) [reply selfdrive/car/gm/interface.py:283]: This isn't doing anything with the CC resume signal, though you are correct that it is noisy. This change would just not show the "press resume" alert if the user is not in fact required to press resume. (Note that currently, no GM cars have working sng on stock; this is more of a preemptive change to the alert for when that becomes a reality.)

**Comment by nworb-cire** [comment]
@adeebshihadeh anything holding this up? I believe it still requires your approval since you requested changes.

**Comment by adeebshihadeh** [comment]
Looks good, thanks for writing the tool too! Also fyi, that's not how the reviews work.

**Pull request merged**
